### ISSUE TITLE
Wolfen auch drüber hosten?

### ISSUE THREAD

**Issue by niccokunzmann** [seth0r/ffp-monitor#3]
Ich möchte eine Community in Bitterfeld-Wolfen aufbauen.
Dabei werden, vielleicht wegen der komischen IPs, keine Links angezeigt, die zwischen den Routern aktiv sind - vielleicht wurde das wegen des VPN rausgenommen.

**Comment by seth0r** [comment]
Ja, das dürfte an den IPs liegen, es gibt einige Klassen an IPs, die alle in ips.py definiert werden.
